socorro/signature/rules.py

```diff
@@ -710,3 +710,43 @@ def action(self, raw_crash, processed_crash, notes):
             raw_crash['IPCMessageName']
         )
         return True
+
+
+class SignatureIPCMessageName(Rule):
```

Comment:
This is a copy-paste of the rule above, maybe you hit paste twice :P

Reply:
I'm having a fucking hell of a time with my keyboard right now. It keeps double-doing things. But I should have caught this before committing.

I'll finish up the work for bug 1377254 (flake8) which would have caught this in CI and local testing.